java/riege.importorder

```diff
@@ -0,0 +1,21 @@
+#Organize Import Order
+#Thu Jul 20 16:55:53 CEST 2023
+11=\#java
+12=\#javax
+13=\#junit
```

Comment:
Isn't jakarta missing at this position?

Reply:
I will double-check with the Checkstyle configuration.